### PR TITLE
fixes #2 and #3 Improved documentation images and updated package.json

### LINKED ISSUE #2: Improve documentation images
Some images in the documentation seem to be squeezed. Also there are images with windows screens. I think Windows is not relevant and does not to be featured. Please only provide screenshots of the browser window and use Linux for larger screenshots.

### PR DESCRIPTION
**Pull request by Manveer695**
Removed windows from images, and the height, width attributes from README.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/heroku/node-js-getting-started"
+    "url": "https://github.com/Manveer695/susi_viberbot"
```
Comment (niranjan94) [inline package.json]: @Manveer695 the repo should be the fossasia repo

**Comment by Manveer695** [comment]
@niranjan94 Ok.

File: package.json (hunk 1 above)
Comment (Manveer695) [inline package.json]: The repository linked to heroku should have my repo's address or fossasia's repo address?

**Comment by heysadboy** [comment]
I think we should put mobile screenshots side by side, in a table. Like this https://github.com/fossasia/open-event-android

**Comment by Manveer695** [comment]
Please let me know if any more changes are required. Thank you.

**Comment by mariobehling** [comment]
Thank you. In future please add the link to the issue in the description. Please read about our best practices in this blog post http://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/

**Pull request merged**